pages/who-we-are/corporate/jobs/information-pack.md

```diff
@@ -1,12 +1,13 @@
 ---
 layout: basic
-title: Job Information Pack
+title: Before you apply
 permalink: /who-we-are/corporate/jobs/job-information-pack/
```

Comment:
Change the permalink to reflect the new title, add a redirect from the existing permalink.